client/socketAddr.js

```diff
@@ -0,0 +1 @@
+window.socketAddr = 'http://localhost:8081';
```

Comment:
as another note, socketAddr.js should be ignored and not pushed to git. I'll have to fix that again in the future